src/codegen.cpp

```diff
@@ -7159,10 +7159,11 @@ extern "C" void jl_init_codegen(void)
     initializeAnalysis(Registry);
 #endif
 
-    InitializeNativeTarget();
-    InitializeNativeTargetAsmPrinter();
-    InitializeNativeTargetAsmParser();
-    InitializeNativeTargetDisassembler();
+    InitializeAllTargets();
```

Comment:
Just FYI, in the case of CUDAnative.jl (external back-end using NVPTX) we initialize [those parts of the backend](https://github.com/JuliaGPU/CUDAnative.jl/blob/bea6eb387bf789c9ee44a3d83597308694b71277/src/jit.jl#L206-L212) we need by ourselves, so for us there's no need to initialize everything eagerly.